### ISSUE TITLE
VoiceOver will not focus on the settings button itself, making it impossible to access for blind and visually impaired users

### ISSUE THREAD

**Issue by santiagoblue7** [signalapp/Signal-iOS#4072]
On the latest beta of Signal for iOS, version 2.32.0.19, as well as the latest version of the app available on the App Store, there is a major accessibility issue when it comes to using the app with the built-in screen reading accessibility feature called VoiceOver on iOS. I, the user, am unable to access the settings button for Signal on the main screen. VoiceOver, the screen reader will not focus on the button itself, making it impossible to access for blind and visually impaired users.
Steps to reproduce:
Step 1: Go to the home screen on an iPhone running iOS 12.1, then open settings, then general, then accessibility, then VoiceOver. Turn on VoiceOver.
Step 2: Using VoiceOver gestures, such as those listed on the VoiceOver screen, go back to the home screen and open Signal. You'll notice that VoiceOver can only detect the current conversations, a search field, and a compose button; however, it cannot find the settings button. 

Here is a link from Apple with resources in terms of developing apps with accessibility in mind. This may contain useful information for fixing this major accessibility bug. [](https://developer.apple.com/accessibility/ios/)

**Comment by santiagoblue7** [comment]
[](url)https://developer.apple.com/accessibility/ios/

**Comment by michaelkirk-signal** [comment]
Thanks for reporting this.

It has been fixed in https://github.com/signalapp/Signal-iOS/pull/4077, which is available in the latest beta (2.32.0.22 or later).

Give it a try and let us know how it goes.

**Comment by santiagoblue7** [comment]
Hello, 
I noticed the bug has been fixed, and Signal is working properly with VoiceOver once again. Thanks for fixing it. 

Santiago
